proto/umee/leverage/v1/tx.proto

```diff
@@ -82,57 +82,75 @@ message MsgRepay {
   // Borrower is the account address repaying a loan and the signer
   // of the message.
   string                   borrower = 1;
-  cosmos.base.v1beta1.Coin asset    = 2 [(gogoproto.nullable) = false];
+  cosmos.base.v1beta1.Coin coin     = 2 [(gogoproto.nullable) = false];
 }
 
 // MsgLiquidate is the request structure for the Liquidate RPC.
 message MsgLiquidate {
   // Liquidator is the account address performing a liquidation and the signer
   // of the message.
-  string                   liquidator   = 1;
+  string liquidator = 1;
   // Borrower is the account whose borrow is being repaid, and collateral consumed,
   // by the liquidation. It does not sign the message.
-  string                   borrower     = 2;
+  string borrower = 2;
   // Repayment is the maximum amount of base tokens that the liquidator is willing
   // to repay.
-  cosmos.base.v1beta1.Coin repayment    = 3 [(gogoproto.nullable) = false];
+  cosmos.base.v1beta1.Coin repayment = 3 [(gogoproto.nullable) = false];
   // RewardDenom is the base token denom that the liquidator is willing to accept
   // as a liquidation reward. The uToken equivalent of any base token rewards
   // will be taken from the borrower's collateral.
-  string                   reward_denom = 4;
+  string reward_denom = 4;
 }
 
 // MsgSupplyResponse defines the Msg/Supply response type.
-message MsgSupplyResponse {}
+message MsgSupplyResponse {
+  // Supplied is the amount of base tokens sent ot the module.
+  cosmos.base.v1beta1.Coin supplied = 1 [(gogoproto.nullable) = false];
```

Comment:
this attribute is not needed - it's a copy of the input (msg.Asset). No need to record it (it's going to be saved in tendermint store)

Reply:
Noted. Removed all input-matching response fields